client/src/containers/interventions/new/component.tsx

```diff
@@ -55,16 +50,20 @@ const STEPS2 = {
 
 const InterventionForm: FC = () => {
   const dispatch = useAppDispatch();
-  const { asPath } = useRouter();
   const { filters } = useAppSelector(analysis);
   const { interventionType } = filters;
+  const { interventions } = useAppSelector(analysis);
+  const { step } = interventions;
 
   const steps = useMemo(() => [STEP1, STEPS2[interventionType]], [interventionType]);
 
-  const currentStep = useMemo<string>(() => asPath.split('#')[1], [asPath]);
-  const isFirstStep = useMemo<boolean>(() => currentStep === 'intervention_step1', [currentStep]);
   const handleCancel = useCallback(() => {
     dispatch(setSubContentCollapsed(true));
+    dispatch(setNewInterventionStep({ id: 'step', value: 1 }));
```

Comment:
```suggestion
    dispatch(setNewInterventionStep(1);
```
you don't need to create an object only for a restricted value